common/src/main/resources/startup.properties

```diff
@@ -348,3 +348,5 @@ it.workflow.execution.listeners=org.apache.falcon.catalog.CatalogPartitionHandle
 ### LogMoveService Properties
 *.falcon.logMoveService.threadCount=200
 *.falcon.logMoveService.blockingQueue.length=50
+*.falcon.logMoveService.CorePoolSize=20
```

Comment:
If we can call these properties falcon.logMoveService.max.threadCount and falcon.logMoveService.min.threadCount, it will be more intuitive to set the values. I understand that ThreadPoolExecutor actually calls it corePoolSize, but, it really is the min.